experimental/Schemes/SpecialTypes.jl

```diff
@@ -16,11 +18,12 @@ complement_equation(U::PrincipalOpenSubset) = U.f::elem_type(OO(ambient_scheme(U
 gens(U::PrincipalOpenSubset) = [lifted_numerator(complement_equation(U))]
 getindex(U::PrincipalOpenSubset, i::Int) = (i == 1 ? U : error("index out of range"))
 
-function inclusion_morphism(U::PrincipalOpenSubset) 
+function inclusion_morphism(U::PrincipalOpenSubset; check::Bool=false) 
   if !isdefined(U, :inc)
     X = ambient_scheme(U)
-    inc = SpecMor(U, X, hom(OO(X), OO(U), gens(OO(U))))
-    U.inc = inc
+    # TODO: Eventually make this an OpenInclusion
+    inc = SpecMor(U, X, hom(OO(X), OO(U), gens(OO(U)), check=check))
+    U.inc = OpenInclusion(inc, ideal(OO(X), complement_equation(U)), check=check)
```

Comment:
Seems like the TODO is done? 
```suggestion
    inc = SpecMor(U, X, hom(OO(X), OO(U), gens(OO(U)), check=check))
    U.inc = OpenInclusion(inc, ideal(OO(X), complement_equation(U)), check=check)
```